src/candis/ios/pipeline/pipeline.py

```diff
@@ -244,224 +288,172 @@ def runner(self, cdat, heap_size = 16384, seed = None, verbose = True, split_per
 
         self.logs.append('Saved ARFF at {path}'.format(path = path))
 
-        JVM.start(max_heap_size = '{size}m'.format(size = heap_size))
-
-        load = Loader(classname = 'weka.core.converters.ArffLoader')
-        # data = load.load_file(path)
-        # save =  Saver(classname = 'weka.core.converters.ArffSaver')
         path = 'diabetes.arff' # For debugging Purposes Only
-        data = load.load_file(os.path.join(head, path)) # For Debugging Purposes Only
-        data.class_is_last() # For Debugging Purposes Only
-        # data.class_index = cdat.iclss
+        dataset = arff.load(open(os.path.join(head, path), 'r')) # For Debugging Purposes Only
```

Comment:
ARFF is a WEKA specific file format that needs to be removed. You have to convert the AffyMetrix File to a PD DataFrame.